gate_detector:
  database: gate.sa

  minimum_contour_area: 10000
  contour_linearization_epsilon: 3.0

  area_difference_thresh: 200000
  color_difference_thresh: 1000

  #This should be less than 0
  minimum_rating: 0.10

  auto_filter_bins: 100
  auto_filter_black_margin: 30
  auto_filter_white_margin: 30
  auto_filter_histogram: false
